quidem-ut/src/test/quidem/org.apache.druid.quidem.QTest/qaJsonCols/funcs_and_sql_func_json_value.03.msq.iq

```diff
@@ -1,5 +1,5 @@
 !set useApproximateCountDistinct false
-!use druidtest://?componentSupplier=StandardMSQComponentSupplier&datasets=sql/src/test/quidem/qatests/qaJsonCols/funcs_and_sql
+!use druidtest://?componentSupplier=StandardComponentSupplier&datasets=sql/src/test/quidem/qatests/qaJsonCols/funcs_and_sql
```

Comment:
This change is wrong: the file name is `.msq.iq` which means it should be using `StandardMSQComponentSupplier`.

Reply:
Sorry, thanks for catching this! I was testing to see if the tests were passing in non-msq engines and accidentally committed this change.